packages/orbit-components/src/Inline/Inline.stories.tsx

```diff
@@ -25,106 +24,91 @@ const TestChildren = () => {
   );
 };
 
-export default {
+const meta: Meta<typeof Inline> = {
   title: "Inline",
+  component: Inline,
+
+  argTypes: {
+    align: {
+      options: Object.values(ALIGN),
+      control: {
+        type: "select",
+      },
+    },
+    justify: {
+      options: Object.values(JUSTIFY),
+      control: {
+        type: "select",
+      },
+    },
+    spacing: {
+      options: Object.values(SPACING),
+      control: {
+        type: "select",
+      },
+    },
+  },
 };
 
-export const Default = ({ spacing }) => {
-  return (
-    <Inline spacing={spacing} dataTest={dataTest}>
+export default meta;
+type Story = StoryObj<typeof Inline>;
+
+export const Default: Story = {
+  render: args => (
+    <Inline {...args}>
       <TestChildren />
     </Inline>
-  );
-};
+  ),
 
-Default.story = {
   parameters: {
     info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
-  },
-};
-
-Default.args = {
-  spacing: SPACING.XXXSMALL,
-};
-
-Default.argTypes = {
-  spacing: {
-    options: Object.values(SPACING),
-    control: {
-      type: "select",
+    controls: {
+      disable: true,
     },
   },
 };
 
-export const Playground = ({ as, align, justify, spacing }) => {
-  return (
-    <Inline as={as} align={align} justify={justify} dataTest={dataTest} spacing={spacing}>
+export const Playground: Story = {
+  render: args => (
+    <Inline {...args}>
       <TestChildren />
     </Inline>
-  );
-};
+  ),
 
-Playground.story = {
   parameters: {
-    info: "info: You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
+    info: "You can try all possible configurations of this component. However, check Orbit.Kiwi for more detailed design guidelines.",
   },
-};
-
-Playground.args = {
-  as: "div",
-  align: ALIGN.START,
-  justify: JUSTIFY.START,
-  spacing: SPACING.XXXSMALL,
-};
 
-Playground.argTypes = {
-  align: {
-    options: Object.values(ALIGN),
-    control: {
-      type: "select",
-    },
-  },
-  justify: {
-    options: Object.values(JUSTIFY),
-    control: {
-      type: "select",
-    },
-  },
-  spacing: {
-    options: Object.values(SPACING),
-    control: {
-      type: "select",
-    },
+  args: {
+    as: "div",
+    spacing: SPACING.XXXSMALL,
+    align: ALIGN.START,
+    justify: JUSTIFY.START,
+    className: "",
+    id: "",
+    mediumMobile: {},
```

Comment:
I would consider to add `example` values. Now these "query" keys are empty objects and it's not clear what to fill at the first moment.

<img width="685" alt="image" src="https://github.com/user-attachments/assets/54a7cf42-901a-45c6-b0e8-dfc3c5823291">

For example, I would change it to:
```
mediumMobile: {
  spacing: SPACING.XXXSMALL,
  align: ALIGN.START,
  justify: JUSTIFY.START,
}
```

WDYT? 🤔

Reply:
I was contemplating adding the examples as you suggest for the same reason but then I didn't because then the other controls (`spacing`, `align` and `justify`) wouldn't work so you would need to change those objects to see the change which seemed too complicated to me 😄 Maybe adding just for one or two and leave the rest empty?

Reply:
yes, I would add at least one/two to demonstrate how it can be filled in. :D